lib/node_modules/@stdlib/complex/float64/conj/package.json

```diff
@@ -14,6 +14,8 @@
     }
   ],
   "main": "./lib",
+  "browser": "./lib/main.js",
```

Comment:
```suggestion
```

Reply:
`native.js` is not loaded through the main entry point `index.js`. As such, this is not necessary.